release/scan-image-vulnerabilities/README.md

```diff
@@ -0,0 +1,112 @@
+# Scan Image Vulnerabilities
+
+Scan a container image on quay.io for vulnerabilities using `roxctl image scan` and fail if fixable critical or important vulnerabilities are found.
+
+This action waits for an image to be available on Quay.io, scans it using roxctl, and generates a detailed vulnerability report in the GitHub step summary, while making the raw report available as JSON in the workspace as `scan-result.json`.
+
+## Required permissions
+
+```yaml
+permissions:
+  # Needed for stackrox/central-login to create the JWT token.
+  id-token: write
+```
+
+## All options
+
+| Input                                     | Description                                        | Required | Default   |
+| ----------------------------------------- | -------------------------------------------------- | -------- | --------- |
+| [image](#image)                           | Image name (without registry prefix)               | Yes      |           |
+| [version](#version)                       | Image version tag                                  | Yes      |           |
+| [wait-limit](#wait-limit)                 | Maximum time to wait for image (seconds)           | No       | `"7200"`  |
+| [summary-prefix](#summary-prefix)         | Title prefix for the GitHub step summary           | Yes      |           |
+| [quay-bearer-token](#quay-bearer-token)   | Quay.io bearer token for wait-for-image            | Yes      |           |
+| [central-url](#central-url)               | ACS Central URL                                    | Yes      |           |
+
+### Detailed options
+
+#### image
+
+Image name without the registry prefix. The action will automatically prepend `quay.io/` to construct the full image reference.
+
+Example: `"rhacs-eng/main"`
+
+Required: Yes
```

Comment:
In the spirit of reducing verbosity and maintenance, I would suggest remove `Required:` mentions here because the table above already has a column for that.
Up to you.